app/src/main/java/org/hisp/dhis/android/dataentry/form/section/viewmodels/FormItemViewModelFactory.java

```diff
@@ -0,0 +1,16 @@
+package org.hisp.dhis.android.dataentry.form.section.viewmodels;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.hisp.dhis.android.core.common.ValueType;
+
+public interface FormItemViewModelFactory {
```

Comment:
I considered both options and decided it was better to have 1 class with 1 responsibility: Creating FormViewModels. We can scatter that logic in the classes that we already have later if you want. I am thinking we keep it as is for now and re-assess after the MVP deadline.